using-etherscan/staking.md

```diff
@@ -0,0 +1,41 @@
+# Staking
+
+Sometimes, the Olympus website might not be accessible due to [hosting issues](https://twitter.com/FleekHQ/status/1416505712222609411). Fear not, you can still interact with the Olympus contracts to perform certain actions such as staking. In this guide, we will show you how to stake OHM tokens via [Etherscan](https://etherscan.io/).
+
+If you have never staked OHM before, there are two steps involved:
+
+1. Approve the staking contract to spend your OHM tokens.
+
+2. Stake your OHM tokens.
+
+If you have staked OHM before, there is only one step to perform: Stake your OHM tokens.
+
+## How to Approve OHM Spending via Etherscan
+
+1. Go to the [Write Contract section of the OHM token contract](https://etherscan.io/address/0x383518188c0c6d7730d91b2c03a03c837814a899#writeContract).
+
+2. Check and ensure your selected network is "Ethereum Mainnet" in your wallet. Then press **Connect to Web3** to connect your wallet if you haven't done so.
+
+3. Once it is connected, select the third option *approve*.
+
+4. On the *spender (address)* field, fill in this value: **0xC8C436271f9A6F10a5B80c8b8eD7D0E8f37a612d**
+
+5. On the *amount (uint256)* field, fill in this value: **1000000000000000000**
```

Comment:
Can we extend this to make clear what value this refers to? Is this 10 or 1000 OHM? Maybe people only want to allow their own values so then they need to understand how many zeroes are required here. It became kind of an anti-pattern to allow the maximum of funds being spendable by contracts.

Reply:
Good suggestion. I have pushed additional commits to explain the value to put into the amount field.